static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -1283,9 +1250,41 @@ class IssueRuleEditor extends AsyncView<Props, State> {
                           </ChevronContainer>
                           <StepContent>
                             <StepLead>
-                              {tct('[then:Then] perform these actions', {
-                                then: <Badge />,
-                              })}
+                              {tct(
+                                '[then:Then] perform these actions once every [selector]',
```

Comment:
this isn't specific enough to how often this alert could trigger cc @robinrendle

Reply:
the 30 minutes is per unique issue